lib/Doctrine/Common/DataFixtures/AbstractFixture.php

```diff
@@ -71,13 +72,27 @@ public function addReference($name, $object)
      *
      * @see Doctrine\Common\DataFixtures\ReferenceRepository::getReference
      *
-     * @param string $name
+     * @param string      $name
+     * @param string|null $class
      *
      * @return object
+     *
+     * @template T of object
+     * @psalm-param class-string<T>|null $class
+     * @psalm-return $class is null ? object : T
      */
-    public function getReference($name)
+    public function getReference($name, $class = null)
     {
-        return $this->referenceRepository->getReference($name);
+        if ($class === null) {
+            Deprecation::trigger(
+                'doctrine/data-fixtures',
+                'https://github.com/doctrine/data-fixtures/pull/409',
+                'Argument $class of %s() will be mandatory in DoctrineDataFixtures 2.0.',
```

Comment:
This `DoctrineDataFixtures` case is typically used for the bundle. I'd remove that name anyway:

```suggestion
                'Argument $class of %s() will be mandatory in 2.0.',
```

Reply:
Done. I also fixed the tests